.gitignore

```
@@ -4,5 +4,5 @@ dist/*
 examples/
 .idea
 .vscode
-.coverage
-venv
\ No newline at end of file
+.coverage*
```

```suggestion
.coverage
.coverage.*
```

probably don't want ignore `.coveragerc`, if we end up using that someday [[1]](https://coverage.readthedocs.io/en/latest/config.html?highlight=.coveragerc)